### ISSUE TITLE
Prepare Display Screens

### ISSUE THREAD

**Issue by mildlyincompetent** [srobo/tasks#76]
Make sure we know what will be running on display screens at the competition, ensure these are up to date and functioning.

**Comment by PeterJCLaw** [comment]
I'm expecting that the display screens will show the usual gamut of arena corner, staging and general information respective to their locations (four in the arena, a couple by the entrance(s) and some large ones generally in the venue).

Unless it's covered elsewhere, it would be useful to also check:
- that we have enough drivers (i.e: probably Pis) for all the screens we want to have
- that we have access to enough actual screens & mounting hardware

**Comment by PeterJCLaw** [comment]
See also #96 for preparing the Pis.

**Comment by Tyler-Ward** [comment]
The screens for the corners are in storage see #183

**Comment by Tyler-Ward** [comment]
we will be getting some pi vesa mounts for Smallpiece that we should consider using at the competition. to make the screen setups easier to setup in one location and then deploy elsewhere.

**Comment by RealOrangeOne** [comment]
Sounds like a good idea! This might seem like a stupid question, but I assume those vesa mounts still allow for the screens to be mounted using vesa? That's quite important for mounting to the scaff.

**Comment by Tyler-Ward** [comment]
not directly however we are also ordering some of these that do https://www.scan.co.uk/products/silverstone-sst-mva01-vesa-extension-bracket-enables-mounting-of-nuc-with-monitor-arm-75x75mm-100x10

**Comment by PeterJCLaw** [comment]
Historically SR has constructed our own mounting situation and it's been fine (folded card works remarkably well, plus you don't need to worry about geting it covered in tape). While I appreciate the desire to level up here, it seems a little odd to do so on a year where we're otherwise trying to be very frugal with our funds.

**Comment by Tyler-Ward** [comment]
I was mostly considering using the ones bought for small-piece rather than buying any more.

**Comment by PeterJCLaw** [comment]
Ah, sorry, I see. If we already have some to hand, then :+1:

**Comment by PeterJCLaw** [comment]
Blocked on the Pis making their way to me for configuration.

**Comment by PeterJCLaw** [comment]
We've worked out a plan for what's going where. The remaining part of this is actually configuring the Pis, which is covered by #96.

**Comment by PeterJCLaw** [comment]
More detail on what's going where:

### L4

- 2 x portrait, by the photo studio. Nice to have.
- overhead as you go down the stairs to L3. Probably an "outside" screen

- large display wall. Has own computer. Showing the livestream.

### L3

- 2 x in SUSU cafe, both VGA:
  - "outside"
  - livestream

- Bar 3 screen "outside"? livestream?
- Cafe by the main door


#### Mezzanine

- compbox
- maybe commentator display

### L2

- Helpdesk "outside" display
- Bar 2 screen "outside" display

- Maybe volunteer display

### Cube

- Arena * 4
- Cube entrance "outside" screen
